src/attr/_make.py

```diff
@@ -1259,6 +1261,7 @@ def fmt_setter_with_converter(attr_name, value_var):
     # This is a dictionary of names to validator and converter callables.
     # Injecting this into __init__ globals lets us avoid lookups.
     names_for_globals = {}
+    annotations = {}
```

Comment:
Do we want to also add the return annotation i.e. `{'return': None}`

Reply:
Yes, please include 'return', too.